src/pages/schema/[...slug].astro

```diff
@@ -0,0 +1,28 @@
+---
+import MainLayout from "../../layouts/MainLayout.astro";
+export async function getStaticPaths() {
+    const schemas = await Astro.glob('../../../schema/generated/*.md');
+    return schemas.map((entry) => {
+        const file = entry.file;
+        let filename = file.split('/').pop();
+        let title = filename.slice(0, -3);
+
+        // Make the generated index.md the root path.
+        if (filename === 'index.md') {
+            filename = undefined;
+            title = 'Schemas';
+        }
+
+        // @TODO: don't keep .md in the slug path.
+        // For now, this makes it easier to navigate while testing.
```

Comment:
Currently these paths include the `.md` extension, eg `/schema/Project.md`. We can remove the file extension but then the relative hyperlinks from the generated markdown files don't work, since they include `.md` extensions. Two potential solutions: 1) modify hyperlinks in markdown files (I don't think there is out-of-the-box support for this) or 2) make redirects in the Astro site to handle `*.md` extensions`?